lib/ansible/galaxy/collection/__init__.py

```diff
@@ -1304,8 +1305,7 @@ def _build_collection_tar(
         file_manifest,  # type: FilesManifestType
 ):  # type: (...) -> str
     """Build a tar.gz collection artifact from the manifest data."""
-    files_manifest_json = to_bytes(json.dumps(file_manifest, indent=True), errors='surrogate_or_strict')
-    collection_manifest['file_manifest_file']['chksum_sha256'] = secure_hash_s(files_manifest_json, hash_func=sha256)
```

Comment:
Removing the hash calculation and assignment is what seems to be failing the CI now.

Reply:
did that work

Reply:
@webknjaz thanks, i was working pretty late at night haha has accidently remove the hash calculation! looks like one of the ubuntu 2004 is for some reason failing on apt : Test update_cache, I dont think its related to the changes  I made.